yaks/spec/unit/yaks/behaviour/optional_includes_spec.rb

```diff
@@ -0,0 +1,63 @@
+require "yaks/behaviour/optional_includes"
+
+RSpec.describe Yaks::Behaviour::OptionalIncludes do
+  include_context 'yaks context'
+
+  subject(:mapper)   { mapper_class.new(yaks_context) }
+  let(:resource)     { mapper.call(instance) }
+
+  let(:mapper_class) do
+    Class.new(Yaks::Mapper).tap do |mapper_class|
+      mapper_class.send :include, Yaks::Behaviour::OptionalIncludes
+      mapper_class.type "user"
+      mapper_class.has_many :posts, mapper: post_mapper_class
+      mapper_class.has_one :account, mapper: account_mapper_class
+    end
```

Comment:
I would, but then `post_mapper_class` and `account_mapper_class` aren't available, because `self` changes. That's the reason why I chose this less pretty approach.